.github/libs/promiseWhile.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line lodash/import-scope
 import type {DebouncedFunc} from 'lodash';
```

Comment:
Unfortunately types imports needs to be ignored manually because `eslint-plugin-lodash` doesn't support types: https://github.com/wix-incubator/eslint-plugin-lodash/issues/337

Reply:
@szymonrybczak this will not be handy as we will work on enabling react-compiler for the entire codebase. Do you think we could fix this in the `eslint-plugin-lodash` upsteam? All files that skip eslint are not processed with react-compiler